### PR TITLE
feat(tui): add German (`de`) locale

### PR DESCRIPTION
**Pull request by fujiapple852**

### CODE REVIEW

File: crates/trippy-tui/locales/app.yml [hunk 1]
```diff
@@ -6,730 +6,835 @@ trippy:
...
   zh: "无"
+  de: "n/a"
 discovered:
   en: "discovered %{hop_count} hops"
   fr: "%{hop_count} sauts découverts"
   tr: "%{hop_count} atlanan keşfedildi"
   it: "%{hop_count} salti trovati"
   pt: "descobriu %{hop_count} saltos"
   zh: "发现 %{hop_count} 跳"
+  de: "entdeckte %{hop_count} hops"
```
Comment (peshay) [inline crates/trippy-tui/locales/app.yml:137]: %{hop_count} gefundene hops

File: crates/trippy-tui/locales/app.yml [hunk 2]
```diff
@@ -6,730 +6,835 @@ trippy:
...
   zh: "扩展"
+  de: "Ext"
 help_tagline:
   en: "A network diagnostic tool"
   fr: "Un outil de diagnostic réseau"
   tr: "Bir ağ analiz aracı"
   it: "Uno strumento diagnostico di rete"
   pt: "Uma ferramenta de diagnóstico de rede"
   zh: "网络诊断工具"
+  de: "Ein netzwerkdiagnosetool"
```
Comment (peshay) [inline crates/trippy-tui/locales/app.yml:465]: Ein Netzwerkdiagnosetool

File: crates/trippy-tui/locales/app.yml [hunk 3]
```diff
@@ -6,730 +6,835 @@ trippy:
...
   zh: "发现 %{hop_count} 跳"
+  de: "entdeckte %{hop_count} hops"
 discovered_flows:
   en: "discovered %{hop_count} hops and %{flow_count} unique %{plural_flows}"
   fr: "%{hop_count} sauts et %{flow_count} %{plural_flows} uniques découverts"
   tr: "%{hop_count} atlanan ve %{flow_count} benzersiz %{plural_flows} keşfedildi"
   it: "scoperti %{hop_count} salti e %{flow_count} %{plural_flows} unici"
   pt: "descobriu %{hop_count} saltos e %{flow_count} %{plural_flows} únicos"
   zh: "发现 %{hop_count} 跳，%{flow_count} 个唯一%{plural_flow}"
+  de: "entdeckte %{hop_count} hops und %{flow_count} einzigartige %{plural_flows}"
```
Comment (peshay) [inline crates/trippy-tui/locales/app.yml:145]: %{hop_count} gefundene hops und %{flow_count} eindeutige %{plural_flows}

**Review by peshay — CHANGES_REQUESTED** [review]

**Comment by fujiapple852** [comment]
Thanks @peshay !

Here is how it looks:

Screenshot 2024-11-06 at 6 35 12 PM

Screenshot 2024-11-06 at 6 35 24 PM

Screenshot 2024-11-06 at 6 35 35 PM

Let me know if any further changes are needed, otherwise this can be merged.

File: crates/trippy-tui/locales/app.yml [hunk 4]
```diff
@@ -599,6 +658,7 @@ help_show_bindings:
   sv: "Tryck på [%{key}] för att visa alla kortkommando"
   ru: "Нажмите [%{key}], чтобы показать все привязки"
   es: "Presiona [%{key}] para mostrar todos los atajos"
+  de: "Drücken Sie [%{key}], um alle Bindungen anzuzeigen"
```
Comment (peshay) [inline crates/trippy-tui/locales/app.yml:661]: I would rather translate this with "Tastaturkürzel" (keyboard shortcuts) or "Tastaturbelegungen" (Keyboard bindings) to make clear that its about key shortcuts.
"Drücken Sie [%{key}], um alle Tastaturkürzel anzuzeigen"

File: crates/trippy-tui/locales/app.yml [hunk 5]
```diff
@@ -729,6 +801,7 @@ settings_tab_bindings_title:
   sv: "Kortkommando"
   ru: "Привязки"
   es: "Atajos"
+  de: "Bindungen"
```
Comment (peshay) [inline crates/trippy-tui/locales/app.yml:804]: I saw somewhere else already "Tastenbelegungen", that would be a good fit.

File: crates/trippy-tui/locales/app.yml [hunk 6]
```diff
@@ -809,6 +889,7 @@ settings_tab_theme_desc:
   sv: "Tui-temafärger"
   ru: "Цвета темы Tui"
   es: "Colores del tema Tui"
+  de: "Tui-Themafarben"
```
Comment (peshay) [inline crates/trippy-tui/locales/app.yml:892]: I would rather use the English "theme" as well here: "Tui-Themefarben"

File: crates/trippy-tui/locales/app.yml [hunk 7]
```diff
@@ -739,6 +812,7 @@ settings_tab_theme_title:
   sv: "Tema"
   ru: "Тема"
   es: "Tema"
+  de: "Thema"
```
Comment (peshay) [inline crates/trippy-tui/locales/app.yml:815]: I would rather use the English "Theme"

File: crates/trippy-tui/locales/app.yml (hunk 4 above)
Comment (peshay) [reply crates/trippy-tui/locales/app.yml:661]: I saw somewhere else already "Tastenbelegungen", that would be a good fit.

File: crates/trippy-tui/locales/app.yml (hunk 7 above)
Comment (peshay) [reply crates/trippy-tui/locales/app.yml:815]: Maybe I would go for "Darstellung".

File: crates/trippy-tui/locales/app.yml (hunk 4 above)
Comment (fujiapple852) [reply crates/trippy-tui/locales/app.yml:661]: So i'm clear then, "Drücken Sie [%{key}], um alle Tastenbelegungen anzuzeigen"?  Should "Tastenbelegungen" be capitalized?

Comment (fujiapple852) [reply crates/trippy-tui/locales/app.yml:661]: @peshay can you confirm on this one?

Comment (peshay) [reply crates/trippy-tui/locales/app.yml:661]: Yes, that is correct. Tastenbelegungen capitalized

**Pull request merged**